test/create-user.test.ts

```diff
@@ -42,7 +42,5 @@ describe('Test createUser', () => {
       email: input.email,
       birthdate: input.birthdate,
     });
-
-    await User.delete({ email: input.email });
```

Comment:
o ideal é que apague mesmo, uma sugestão seria você utilizar um [afterEach](https://mochajs.org/#hooks) para garantir que após cada teste (cada it) vai rolar uma limpa no banco

Reply:
Verdade, eu acabei encadeando os testes para não ter que inserir um usuário em todo teste, mas entendi os problemas relacionados à isso. Corrigido, obrigado!